eclipse_development_process.adoc

```diff
@@ -310,182 +310,180 @@ digraph {
 
 [#6_4_Releases]
 === 6.4 Releases
 
-Any project, with exception of permanent incubators, may make a release. A release may include the code from any subset of the project's descendants.
+Any Project, with exception of Permanent Incubators, may make a Release. A Release may include the code from any subset of the Project's descendants.
 
-_(Most of this section is paraphrased from the excellent http://www.apache.org/dev/release.html[Apache Software Foundation Releases FAQ]. The Eclipse community has many of the same beliefs about Releases as does the Apache community and their words were already excellent. The Apache Software Foundation Releases FAQ is distributed under the http://www.apache.org/licenses/LICENSE-2.0[Apache License, Version 2.0].)_
+_(Most of this section is paraphrased from the excellent http://www.apache.org/dev/Release.html[Apache Software Foundation Releases FAQ]. The Eclipse community has many of the same beliefs about Releases as does the Apache community and their words were already excellent. The Apache Software Foundation Releases FAQ is distributed under the http://www.apache.org/licenses/LICENSE-2.0[Apache License, Version 2.0].)_
 
-Releases are, by definition, anything that is distributed outside of the committers of a project. If consumers are being directed to download a build, then that build has been released (modulo the exceptions below). All projects and committers must obey the Eclipse Foundation requirements on approving any release.
+Releases are, by definition, anything that is distributed outside of the Committers of a Project. If consumers are being directed to download a build, then that build has been Released (modulo the exceptions below). All Projects and Committers must obey the Eclipse Foundation requirements on approving any Release.
 
-The {ipPolicyUrl}[Eclipse IP Policy] must be followed at all times. Prior to issuing any release, the Project Lead must assert that the IP Policy has been followed and all approvals have been received.
+The {ipPolicyUrl}[Eclipse IP Policy] must be followed at all times. Prior to issuing any Release, the Project Lead must assert that the IP Policy has been followed and all approvals have been received.
 
-A project can make official releases for one calendar year following a successful link:#6_3_5_Progress_Review[progress review] or link:#6_3_3_Release_Review[release review]. The project leadership chain may--at its discretion--require that the project engage in additional reviews (e.g. a progress or release review) prior to issuing a release.
+The Project Team must provide (either directly or indirectly) a link between the distribution form of Release artifacts and the corresponding source code.
```

Comment:
I wonder if `(either directly or indirectly)` is necessary. Should we just omit it?

Reply:
I'm going to leave it for now. I would like to avoid folks thinking that direct links in compiled content are required.